mojito-functions/reports.R

```diff
@@ -55,28 +57,9 @@ mojitoReportExperimentSizing <- function(conversion_point) {
   }
 }
 
-
-# SRM test
-# Check for assignment issues with Ron Kohavi's SRM check: https://twitter.com/ronnyk/status/932798952679776256?lang=en
-mojitoSrmTest <- function(wave_params, dailyDf, expected_ratio=0.5) {
```

Comment:
Any particular reason to implement this test from scratch instead of using the built-in `chisq.test`?

```
observed = c(1000,950,1050,1000) # observed frequencies
expected = c(.25,.25,.25,.25) # expected proportions

chisqr = chisq.test(x = observed, p = expected) # chi-square test

chisqr$p.value # p-value of chi-square test
```

Should output something close to 0.1717971 just like [this unit test](https://github.com/lukasvermeer/srm/blob/master/chrome-extension/tests/computeSRM.test.js) and [this online calculator](https://www.gigacalculator.com/calculators/chi-square-calculator.php?test=goodnessoffit&data=1000+0.25%0D%0A950+0.25%0D%0A1050+0.25%0D%0A1000+0.25).

Reply:
Thanks for checking out our repo @lukasvermeer!

It's embarrassingly old code I translated from an old tweet by Ron Kohavi (as we were just learning about SRM tests). We're doing some refactoring and tidying on this repo to address this though.

The `chisq.test()` function looks more appropriate here than our `binom.test()` approach - really helpful!

Reply:
Looks like your test is reproducible here for `chisq.test()` too - working across our reports here too:

```
> wave_params <- list(
+   recipes=c("Control","Treatment1","Treatment2","Treatment3")
+ )
> df <- data.frame(recipe_name=wave_params$recipes, exposure_time=c("2020-02-04","2020-02-04","2020-02-04","2020-02-04"),subjects=c(1000,950,1050,1000))
> mojitoDiagnostics(wave_params, df, proportions = c(0.25,0.25,0.25,0.25))
SRM p-value: 0.172
Control: 25.0% (25.0% expected)
Treatment1: 23.7% (25.0% expected)
Treatment2: 26.2% (25.0% expected)
Treatment3: 25.0% (25.0% expected)
```

Reply:
Made some updates to use `chisq.test()` as you suggested: 

https://github.com/mint-metrics/mojito-r-analytics/pull/2/files#diff-a3159abb590ecbf27d77e871593408c7R93-R108

Reply:
`binom.test` would also work, I think, but only for A/B, not A/B/n.

Reply:
Yep - `binom.test` wouldn't give us what we need for A/B/n... Thanks again!